### ISSUE TITLE
Json Error when running this application

### ISSUE THREAD

**Issue by gelsas** [jyggen/intro-detection-info#2]
Hi I am getting the following error when running your script:

```
./intro-detection-info http://myServerIPHere:32400/ PLEXTOKEN
⠴ Fetching metadata... Error: json: cannot unmarshal array into Go struct field Metadata.MediaContainer.Metadata.guid of type string

```

What can I do to make it work ?

**Comment by fmr-iv** [comment]
@jyggen

**Comment by jyggen** [comment]
This issue should now be solved in 1.0.1. Sorry for the delay, this issue completely flew under my radar.
